api/database/migrations/2021_08_06_153938_update_users_with_roles.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateUsersWithRoles extends Migration
```

Comment:
Consider renaming the class `UpdateUsersWithRoles` to `UpdateUsersTableWithRoles` (and the migration filename from `2021_08_06_153938_update_users_with_roles.php` to `2021_08_06_153938_update_users_table_with_roles.php`) to match convention for other migrations that include the word **table**.